packages/playground/website/src/github/github-export-form/modal.tsx

```diff
@@ -1,49 +1,36 @@
-import { signal } from '@preact/signals-react';
-
 import Modal, { defaultStyles } from '../../components/modal';
 import GitHubExportForm, { GitHubExportFormProps } from './form';
 import { usePlaygroundClient } from '../../lib/use-playground-client';
-
-const query = new URLSearchParams(window.location.search);
-export const isGitHubExportModalOpen = signal(
-	query.get('state') === 'github-export'
```

Comment:
Note that removing this signal means we need to move the default modal initialization to `slice-ui.ts`:

https://github.com/WordPress/wordpress-playground/blob/9493096a5e955891329928f5fb1b3027021d04c3/packages/playground/website/src/lib/state/redux/slice-ui.ts#L31

Without porting this line, the modal won't re-appear after refreshing the page (or returning from the OAuth flow).

Reply:
I was wondering if we use this functionality of opening on a parameter from query somewhere. Thanks!